cmd/recommend.go

```diff
@@ -50,4 +50,6 @@ func init() {
 	recommendCmd.Flags().StringVarP(&recommendOptions.ReportFile, "report", "r", "report.txt", "report file")
 	recommendCmd.Flags().StringSliceVarP(&recommendOptions.Tags, "tag", "t", []string{}, "tags (comma-separated) to apply. Eg. PCI-DSS, MITRE")
 	recommendCmd.Flags().StringVarP(&recommendOptions.Config, "config", "c", recommend.UserHome()+"/.docker/config.json", "absolute path to image registry configuration file")
+	recommendCmd.Flags().StringVarP(&recommendOptions.Merge, "merge", "m", "merged-policies.yaml", "Merge hardening policies")
```

Comment:
@yasin-cs-ko-ak can we have the merge flag should be boolean with default value as false

Reply:
thank you for your thorough review and Changes are updated.